nncf/config_schema.py

```diff
@@ -247,6 +247,12 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
                                                              "algorithm. It's calculated  between mean average trace "
                                                              "from previous iteration and current one. 1e-5 by default"
                                                              "bitwidth_per_scope"),
+                    "compression_ratio": with_attributes(_NUMBER,
```

Comment:
I am ok with the name the description is not clear to me. What does `Minimum compression ratio`? I would rephrase that this is a ratio between bits complexity of fully INT8 model and mixed-precision lower-bit model. Minimum requires additional explanation.

Reply:
I would use the term "Bit complexity" everywhere below as well.